src/main/java/liquibase/ext/spanner/sqlgenerator/CreateSequenceGeneratorSpanner.java

```diff
@@ -52,12 +50,6 @@ public Sql[] generateSql(CreateSequenceStatement statement, Database database,
     queryStringBuilder.append("CREATE SEQUENCE ");
     queryStringBuilder.append(database.escapeSequenceName(statement.getCatalogName(), statement.getSchemaName(), statement.getSequenceName()));
     queryStringBuilder.append(" OPTIONS (sequence_kind='bit_reversed_positive'");
-    if (statement.getMinValue() != null) {
-      queryStringBuilder.append(", skip_range_min = ").append(statement.getMinValue());
-    }
-    if (statement.getMaxValue() != null) {
-      queryStringBuilder.append(", skip_range_max = ").append(statement.getMaxValue());
-    }
```

Comment:
If this was already there, then we should not remove it. That would be a breaking change. That also means that we should remove the additional disallowedFields above.

Reply:
Rollback and  remove the additional disallowedFields above.

Reply:
should also restore minValue and maxValue in  changelog create-sequence.spanner.yaml?